docs/runners/kubernetes.md

```diff
@@ -0,0 +1,150 @@
+# Kubernetes Runner
+
+The Kubernetes Runner runs a MLCommons-Box on a Kubernetes cluster.
+
+[Skip over to the fun part.](#prepare-a-kubernetes-cluster)
+
+## Why Kubernetes?
+
+One of the key goals of the MLCommons-Box project is to enable portability of ML models.
+Kubernetes offers the a good set of abstractions to enable model training to be portable
+across different compute platforms.
+
+## Design
+
+[Kubernetes Runner Proposal Doc](http://bit.ly/box-k8s-runner)
+
+The Kubernetes runner takes in a kubernetes specific task file in the `run` directory and re-uses the Docker runner
+platform config and prepares a Kubernetes Job manifest. The runner then creates the job on the Kubernetes cluster.
+
+![Design](../assets/mlcommons-box-k8s.png)
+
+
+The Kubernetes Runner takes in a MLBox run configuration file similar to other runners. With clear definitions of input
+and output bindings.
+Here's an example:
+
+```yaml
+schema_type: mlbox_invoke
+schema_version: 1.0.0
+
+task_name: kubernetes   # task name set to 'kubernetes'
+
+input_binding:  # input parameters (name: value)
+  data_dir:
+    path: workspace/data
+    k8s:
+      persistentVolumeClaim: mlbox-input
```

Comment:
how about using snake_case here to make it consistent with the rest of the config file? e.g. "persistent_volume_claim" or just "pvc"

Reply:
changed it to pvc